charts/karpenter/templates/rbac.yaml

```diff
@@ -43,6 +43,13 @@ metadata:
   name: karpenter-leader-election
   namespace: {{ .Release.Namespace }}
 rules:
+- apiGroups:
+  - "coordination.k8s.io"
```

Comment:
why is this change necessary?

Reply:
I added this as a response to an error locally where I accidentally installed Karpenter to the default namespace, my bad.